client/redux-store.html

```diff
@@ -0,0 +1,38 @@
+<link rel="import" href="/../bower_components/polymer-redux/polymer-redux.html">
+<script src="/../node_modules/redux/dist/redux.js"></script>
+
+<!--
+ This element creates and describes the redux store.
+ ---
+ Redux introduction: http://redux.js.org/docs/introduction/
+ Polymer-redux docs: https://tur-nr.github.io/polymer-redux/
+ -->
+<script>
+    const initialState = {
+        currentLocation: 'homepage',
```

Comment:
What if the first page I visit is 'donation page' or anything other than homepage? For example from the link sent by someone to me.

Reply:
The basic process looks like this: 

the URL changes -> <sc-page-selector> detects this and updates the state -> the correct view is rendered.

For this case it's the same. The value is there to provide an idea of how the initial state tree would look, but it gets re-evaluated on every route change anyway.